src/Persisters/BasicEntityPersister.php

```diff
@@ -150,6 +150,15 @@ public function getMatchCypher(array $criteria = [], $limit = null, $offset = nu
 
         $cypher .= 'RETURN '.$identifier;
 
+        if (is_array($orderBy) && count($orderBy) > 0) {
+            $cypher .= PHP_EOL;
+            $i = 0;
+            foreach ($orderBy as $property => $order) {
+                $cypher .= $i === 0 ? 'ORDER BY ' : ', ';
```

Comment:
Should be `$cypher .= $i++ === 0 `

Or add `$i++` on a new row for readability.